debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/DumpModuleReadsCommand.java

```diff
@@ -40,21 +40,21 @@
 import com.ibm.j9ddr.vm29.tools.ddrinteractive.JavaVersionHelper;
 
 /**
- * DumpModuleReads command displays all modules that the target module reads
+ * DumpModuleReads command displays all modules that require the target module
  * 
  * Example:
  * !dumpmodulereads 0x00000130550C7E88
  * Example output: 
  * !j9module 0x000001305F46DB38
  * !j9module 0x00000130550DBB58
  * !j9module 0x000001305F469EA8
- * .....(A list of every module read by the target module)
+ * .....(A list of every module that requires the target module)
```

Comment:
If this command is about querying which modules are 'required' by a particular module, then the class and the name of the command it implements should also be adjusted. Likewise for `FindAllReadsCommand`.

Reply:
Do you want me to maybe change their names to  `dumpmodulerequires` and `dumprequiresmodule`?

Reply:
@keithc-ca ?

Reply:
I spoke with @Ailloviee and @tajila suggesting that we should have an issue that captures a plan for the set of new module-related DDR commands so we can be avoid changing command names (more than once). I haven't seen that issue yet and so it's unclear (to me) what the final names should be.